Core/Config.cpp

```diff
@@ -352,7 +352,7 @@ static ConfigSetting generalSettings[] = {
 	ConfigSetting("ForceLagSync", &g_Config.bForceLagSync, false, true, true),
 
 	ReportedConfigSetting("NumWorkerThreads", &g_Config.iNumWorkerThreads, &DefaultNumWorkers, true, true),
-	ConfigSetting("EnableAutoLoad", &g_Config.bEnableAutoLoad, false, true, true),
+	ConfigSetting("EnableAutoLoad", &g_Config.iEnableAutoLoad, false, true, true),
```

Comment:
The default should be 0, rather than false, now.  It would be preferable to add an enum to ConfigValues.h assuming #11187 gets merged.

Also, since we're changing the value anyway, maybe "AutoLoadSaveState" or something is a better name.  AutoLoad makes me think about auto loading a game (and auto loading the last played game is not a totally crazy feature...)

-[Unknown]

Reply:
Ah yes of course, misunderstood the prototype there.  I've gone the enum way as suggested.